README.md

```diff
@@ -64,6 +64,17 @@ Additionally PgSanity will read SQL from stdin if it is not given any parameters
     $ echo $?
     0
 
+# # pre-commit
```

Comment:
Please fix the markdown syntax here: "# #" -> "##"

Reply:
@markdrago 

Fixed. Thanks!